packages/core/src/routes/verification/index.ts

```diff
@@ -81,24 +81,32 @@ export default function verificationRoutes<T extends UserRouter>(
     koaGuard({
       body: z.object({
         identifier: verificationCodeIdentifierGuard,
+        // Optional flag: use BindMfa template type when sending code
+        forBindMfa: z.boolean().optional(),
```

Comment:
How about just allowing developers to specify the template type? Would it be more straightforward?

Reply:
updated